app/controllers/patients_controller.rb

```diff
@@ -281,6 +283,7 @@ def bulk_update_status
 
     # Nothing to do in this function if there isn't a list of patient ids.
     patient_ids = params.require(:ids)
+    non_dependent_patient_ids = patient_ids
```

Comment:
These are the same IDs of patients that we iterate through below it looks like, so how would `non_dependent_patient_ids.include?(patient[:id])` ever be false here?

Reply:
it becomes false for dependents because those dependent ids are added to patient_ids: https://github.com/SaraAlert/SaraAlert/pull/371/files/3b8d6282356126fff346b878902747a106eeb5d5#diff-4f4c29154577c3b2ecf2075ade80b28eR304

Reply:
Ah,  I missed line 302. Got it.